.ci/semaphore.answers.sh

```diff
@@ -5,7 +5,7 @@ pull_and_build() {
     docker build -t meetmangukiya/corobo-answers answers/
 }
 
-if [[ $BRANCH_NAME != "master" ]]
+if [[ $BRANCH_NAME != "master" && $SEMAPHORE_REPO_SLUG != "coala/corobo" ]]
```

Comment:
The second one was not breaking the build. It was supoosed to be run only when the branch is master. So I assumed it was the case of direct push and hence it will be coala/corobo. Also not being the master branch means it's a PR and not coala/corobo that's why I put the condition there.

Reply:
Also I think check_docker.py and pull_and_build will still run if it a fork.

Reply:
The issue is about "deploy-preparation steps".  The task is to prevent build breakages which are occurring.

`.ci/check_docker.py` & `pull_and_build` do not causes breakages.  They are not being run on master because it is assumed that the tests have already been run on the PR.  If it is in master, it was tested already.  So it doesnt need to be done again.

`docker push` creates breakages, because random forkers and PRs dont have permission to push to that docker image name.  That is the problem which needs fixing.